docs/migration.md

```diff
@@ -0,0 +1,181 @@
+
+Migration Guide
+==============================================================================
+
+This guide provides instruction for upgrading your test suite from the 
+[Legacy APIs](legacy.md) to Ember's latest testing APIs based on RFCs
+[232](https://github.com/emberjs/rfcs/blob/master/text/0232-simplify-qunit-testing-api.md)
+and
+[268](https://github.com/emberjs/rfcs/blob/master/text/0268-acceptance-testing-refactor.md).
+
+Upgrading to the new testing APIs
+------------------------------------------------------------------------------
+
+For the complete introduction to the new testing APIs, please read the
+latest [Ember Guides](https://guides.emberjs.com/v3.0.0/testing/). The
+following examples will give you an overview how to migrate your existing Ember
+QUnit based test suite.
+
+### Unit tests
+
+Before:
+
+```javascript
+import { test, moduleFor } from 'ember-qunit';
+
+moduleFor('controller:posts', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+});
+
+// Replace this with your real tests.
+test('exists', function(assert) {
+  let controller = this.subject();
+  assert.ok(controller);
+});
+```
+
+After:
+
+```javascript
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('SidebarController', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('exists', function() {
+    let controller = this.owner.lookup('controller:sidebar');
```

Comment:
@simonihmig I just noticed that this is using a different controller name than the above snippet

Reply:
Oh, indeed. 🙄 Will create a PR...